lisa/tools/mount.py

```diff
@@ -57,15 +64,17 @@ class Mount(Tool):
     # /dev/sda1 on / type ext4 (rw,relatime,discard)
     # /dev/sda1 on /mnt/a type ext4 (rw,relatime,discard)
     _partition_info_regex = re.compile(
-        r"\s*/dev/(?P<name>.*)\s+on\s+(?P<mount_point>.*)\s+type\s+(?P<type>.*)\s+.*"
+        r"\s*/dev/(?P<name>.*)\s+on\s+(?P<mount_point>.*)\s+type"
+        r"\s+(?P<type>.*)\s+\(?(?P<options>.*)\)?"
     )
     _mount_info_regex = re.compile(
         r"\s*(?P<name>.*)\s+on\s+(?P<mount_point>.*)\s+type\s+(?P<type>.*)\s\(+.*"
     )
 
     # /dev/da1p1 on /mnt/resource (ufs, local, soft-updates)
     _partition_info_regex_bsd = re.compile(
-        r"\s*/dev/(?P<name>.*)\s+on\s+(?P<mount_point>.*)\s+(\((?P<type>.*),.*,.*\))"
+        r"\s*/dev/(?P<name>.*)\s+on\s+(?P<mount_point>.*)\s+"
+        r"(\((?P<type>.*),(?P<options>.*,.*)\))"
```

Comment:
Should the "options" be "local" in the example? Is it expected?

Reply:
I'm assuming BSD partitions can contain more than three options in the tuple, similar to Linux partitions.

Reply:
So the pattern should grab the fs type; the first option, then grab the rest as generic 'options'

Reply:
To capture all rest, it's better to end with `(?P<options>.*)\))` . It handles more than 2 options.